include/zephyr/sys/util.h

```diff
@@ -259,6 +259,24 @@ extern "C" {
  */
 #define DIV_ROUND_UP(n, d) (((n) + (d) - 1) / (d))
 
+#ifndef DIV_ROUND_NEAREST
```

Comment:
if defined somewhere else it's their problem

ref  https://docs.zephyrproject.org/latest/contribute/coding_guidelines/index.html#rule-a-3-macro-name-collisions

Reply:
Done.
BTW some macros in this file (MAX, MIN, CLAMP) are guarded with "ifndef", while others are not.
We might want to remove the guards for the sake of consistency.